### ISSUE TITLE
test-controller activity-hash check is too sensitive

### ISSUE THREAD

**Issue by warner** [Agoric/agoric-sdk#3718]
## Describe the bug

@kriskowal made a minor change to some code (probably `assert`) which gets folded into one of the built-in vats , and found that `test-controller.js` started to fail.

As part of #3442 , we fold kernel state into this "activity hash", and that state includes the vat bundles for all the built-in vats, such as comms and vatAdmin. These vats, in turn, are built by calling `bundleSource` on code from the swingset package, but the bundling process pulls in a number of external libraries (including `marshal` and `assert`). So any changes in those upstream libraries will cause the initial activity hash to be different.

Basically, the test is too sensitive. The plan is to downgrade it to merely assert that `controller.getActivityhash()` works at all, and returns a hash-shaped string, but not assert the actual value.
